[
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.iAx6lopd1G"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:14 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, triggers, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_triggers&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;triggers&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_triggers__&quot; ON &quot;mo_user_sync_triggers&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, mo_user_sync_triggers__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(55): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(114): DBUtils->mo_user_sync_create_triggers()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_create_triggers, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:14 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_name, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_value, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_type, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_remote_server_option&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;option_name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_value&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_type&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT 'NULL' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_remote_server_option__&quot; ON &quot;mo_user_sync_remote_server_option&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, mo_user_sync_remote_server_option__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(75): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(115): DBUtils->mo_user_sync_remote_server_option()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_remote_server_option, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, triggers, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_triggers&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;triggers&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_triggers__&quot; ON &quot;mo_user_sync_triggers&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, mo_user_sync_triggers__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(55): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(114): DBUtils->mo_user_sync_create_triggers()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_create_triggers, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_name, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_value, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_type, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_remote_server_option&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;option_name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_value&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_type&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT 'NULL' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_remote_server_option__&quot; ON &quot;mo_user_sync_remote_server_option&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, mo_user_sync_remote_server_option__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(75): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(115): DBUtils->mo_user_sync_remote_server_option()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_remote_server_option, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:33 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:42 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:52 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322",
    "level": "FATAL",
    "type": "PHP",
    "test": "ast-sqlite-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:25:38 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322\nStack trace:\n#0 /wordpress/wp-content/plugins/users-ultra/xoousers.php(56): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('users-ultra/xoo...')\n#3 /internal/eval.php(20): activate_plugin('users-ultra/xoo...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/users-ultra/functions/functions.php on line 322\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #5 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"users-ultra\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.aW3EHkoZgc"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:14 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, triggers, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_triggers&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;triggers&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_triggers__&quot; ON &quot;mo_user_sync_triggers&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, mo_user_sync_triggers__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(55): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(114): DBUtils->mo_user_sync_create_triggers()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_create_triggers, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:14 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_name, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_value, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_type, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_remote_server_option&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;option_name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_value&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_type&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT 'NULL' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_remote_server_option__&quot; ON &quot;mo_user_sync_remote_server_option&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, mo_user_sync_remote_server_option__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(75): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(115): DBUtils->mo_user_sync_remote_server_option()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_remote_server_option, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, triggers, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_triggers&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;triggers&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_triggers__&quot; ON &quot;mo_user_sync_triggers&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, mo_user_sync_triggers__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(55): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(114): DBUtils->mo_user_sync_create_triggers()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_create_triggers, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_name, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_value, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_type, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_remote_server_option&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;option_name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_value&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_type&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT 'NULL' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_remote_server_option__&quot; ON &quot;mo_user_sync_remote_server_option&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, mo_user_sync_remote_server_option__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(75): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(115): DBUtils->mo_user_sync_remote_server_option()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_remote_server_option, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:33 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:42 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:52 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:25:38 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322\nStack trace:\n#0 /wordpress/wp-content/plugins/users-ultra/xoousers.php(56): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('users-ultra/xoo...')\n#3 /internal/eval.php(20): activate_plugin('users-ultra/xoo...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/users-ultra/functions/functions.php on line 322",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322",
    "level": "FATAL",
    "type": "PHP",
    "test": "asyncify-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:25:49 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322\nStack trace:\n#0 /wordpress/wp-content/plugins/users-ultra/xoousers.php(56): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('users-ultra/xoo...')\n#3 /internal/eval.php(20): activate_plugin('users-ultra/xoo...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/users-ultra/functions/functions.php on line 322\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"users-ultra\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "level": "FATAL",
    "type": "PLAYGROUND",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n",
    "log": "/tmp/tmp.YfZ6uOfF8J"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:14 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, triggers, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_triggers&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;triggers&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_triggers__&quot; ON &quot;mo_user_sync_triggers&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, mo_user_sync_triggers__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(55): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(114): DBUtils->mo_user_sync_create_triggers()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_create_triggers, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:14 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_name, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_value, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_type, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_remote_server_option&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;option_name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_value&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_type&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT 'NULL' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_remote_server_option__&quot; ON &quot;mo_user_sync_remote_server_option&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, mo_user_sync_remote_server_option__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(75): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(115): DBUtils->mo_user_sync_remote_server_option()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_remote_server_option, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, triggers, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_triggers&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;triggers&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_triggers__&quot; ON &quot;mo_user_sync_triggers&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_triggers, mo_user_sync_triggers__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(55): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(114): DBUtils->mo_user_sync_create_triggers()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_triggers (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `triggers` text NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_create_triggers, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.",
    "level": "FATAL",
    "type": "SQL",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:23:24 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, remote_server_id, bigint(20) unsigned</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_name, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_value, text</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, option_type, text</li>\n<li>Executing: CREATE TABLE &quot;mo_user_sync_remote_server_option&quot; (\n&quot;id&quot; integer PRIMARY KEY AUTOINCREMENT NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;remote_server_id&quot; integer NOT NULL ON CONFLICT REPLACE DEFAULT 0,\n&quot;option_name&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_value&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT '' COLLATE NOCASE,\n&quot;option_type&quot; text NOT NULL ON CONFLICT REPLACE DEFAULT 'NULL' COLLATE NOCASE) | (no parameters)</li>\n<li>Executing: CREATE  INDEX  &quot;mo_user_sync_remote_server_option__&quot; ON &quot;mo_user_sync_remote_server_option&quot; (&quot;&quot;) | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: mo_user_sync_remote_server_option, mo_user_sync_remote_server_option__, </li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: _mysql_data_types_cache.mysql_type.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE mo...')\n#2 /wordpress/wp-content/plugins/user-sync/Utilities/mo-user-sync-DBUtils.php(75): dbDelta(Array, true)\n#3 /wordpress/wp-content/plugins/user-sync/mo-user-sync-main.php(115): DBUtils->mo_user_sync_remote_server_option()\n#4 /wordpress/wp-includes/class-wp-hook.php(324): mo_user_sync_main->mo_user_sync_plugin_init(false)\n#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_user-s...', false)\n#8 /internal/eval.php(20): activate_plugin('user-sync/mo-us...')\n#9 {main}</pre>\n for query CREATE TABLE mo_user_sync_remote_server_option (\n            `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,\n            `remote_server_id` bigint(20) unsigned NOT NULL,\n            `option_name` text NOT NULL,\n            `option_value` text NOT NULL,\n            `option_type` text DEFAULT 'NULL' NOT NULL,\n            PRIMARY KEY (id),\n            FOREIGN KEY (remote_server_id) REFERENCES mo_user_sync_remote_server_list(id)\n            ON DELETE CASCADE ) ENGINE=InnoDB DEFAULT CHARACTER SET utf8mb4 made by activate_plugin, do_action('activate_user-sync/mo-user-sync-main.php'), WP_Hook->do_action, WP_Hook->apply_filters, mo_user_sync_main->mo_user_sync_plugin_init, DBUtils->mo_user_sync_remote_server_option, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:33 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:42 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "level": "WARNING",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:24:52 UTC] PHP Warning:  Undefined property: stdClass::$ID in /wordpress/wp-content/plugins/usermap/userMap.php on line 49",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:25:38 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322\nStack trace:\n#0 /wordpress/wp-content/plugins/users-ultra/xoousers.php(56): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('users-ultra/xoo...')\n#3 /internal/eval.php(20): activate_plugin('users-ultra/xoo...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/users-ultra/functions/functions.php on line 322",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:25:49 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322\nStack trace:\n#0 /wordpress/wp-content/plugins/users-ultra/xoousers.php(56): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('users-ultra/xoo...')\n#3 /internal/eval.php(20): activate_plugin('users-ultra/xoo...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/users-ultra/functions/functions.php on line 322",
    "log": "/logs/plugins/u/users-ultra/error.json"
  },
  {
    "message": "Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322",
    "level": "FATAL",
    "type": "PHP",
    "test": "jspi-boot",
    "plugin": "users-ultra",
    "details": "[01-May-2025 19:26:00 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /wordpress/wp-content/plugins/users-ultra/functions/functions.php:322\nStack trace:\n#0 /wordpress/wp-content/plugins/users-ultra/xoousers.php(56): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('users-ultra/xoo...')\n#3 /internal/eval.php(20): activate_plugin('users-ultra/xoo...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/users-ultra/functions/functions.php on line 322\n\n    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)\n    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n  [cause]: Error: Error when executing the blueprint step #1 ({\"step\":\"installPlugin\",\"pluginData\":{\"resource\":\"wordpress.org/plugins\",\"slug\":\"users-ultra\"},\"options\":{\"activate\":true}}) : PHP.run() failed with exit code 255 and the following output: \n      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n      ... 2 lines matching cause stack trace ...\n      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: \n        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)\n        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)\n        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)\n        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)\n        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)\n        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)\n        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)\n        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {\n      response: PHPResponse {\n        httpStatusCode: 500,\n        headers: {\n          'x-powered-by': [ 'PHP/8.0.30-dev' ],\n          'content-type': [ 'text/html; charset=UTF-8' ]\n        },\n        bytes: Uint8Array(0) [],\n        exitCode: 255,\n        errors: ''\n      },\n      source: 'request'\n    }\n  }\n}\n\n",
    "log": "/logs/plugins/u/users-ultra/error.json"
  }
]
